server_tests/test/clientTests.js

```diff
@@ -831,7 +837,10 @@ describe("RUST API TEST SUITE", function() {
 
         it("Expect success. camera.getOptions gets correct options when gettable options are set to supported values", function() {
             return testClient.getOptions(sessionId, specifiedOptions)
-            .then( (res) => validate.done(res.body, schema.names.commandGetOptions),
+            .then( (res) => {validate.done(res.body, schema.names.commandGetOptions);
+                for(var i = 0; i < specifiedOptions.length; i++) {
+                    assert.deepEqual(res.body.results.options[specifiedOptions[i]], specifiedOptions[i]);
+                }},
```

Comment:
Curly braces here are a bit compact and inconsistent with other inline functions.